app/assets/stylesheets/views/_history-people.scss

```diff
@@ -60,74 +58,78 @@
 
 
 .historic-people-show {
+  $gutter: 30px;
+  $grey: darken($govuk-secondary-text-colour, 2%);
+
   .person-info {
-    @include media(tablet) {
+    @include govuk-media-query($until: tablet) {
       float: left;
-      width: $one-quarter;
+      width: 25%;
     }
   }
 
   .person-info {
-
-    .name-title, .person-detail, .person-profile {
+    .name-title,
+    .person-detail,
+    .person-profile {
       .inner {
-        padding: 0 $gutter-half;
+        padding: 0 $gutter / 2;
       }
     }
 
     .person-profile {
-      @include core-14;
       width: 218px;
 
-      img {
-        max-width: $full-width;
-      }
       .info {
-        background-color: $grey-3;
-        padding-bottom: $gutter-half;
+        background-color: $grey;
```

Comment:
We should be able to use a colour from the design system. Perhaps `$govuk-secondary-text-colour`?